.ci_support/linux_64_python3.10.____cpython.yaml

```diff
@@ -11,49 +11,39 @@ channel_targets:
 cxx_compiler:
 - gxx
 cxx_compiler_version:
-- '9'
+- '10'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 exiv2:
 - '0.27'
 expat:
 - '2'
 gdal:
-- '3.2'
+- '3.5'
 geos:
-- '3.9'
+- 3.11.0
 gsl:
 - '2.7'
 icu:
-- '68'
+- '70'
+libgdal:
+- '3.5'
 libprotobuf:
-- '3.18'
+- '3.21'
 libspatialindex:
 - 1.9.3
 libzip:
 - '1'
 pin_run_as_build:
-  gdal:
-    max_pin: x.x
   python:
     min_pin: x.x
     max_pin: x.x
-  qt:
-    max_pin: x.x
-  qtkeychain:
-    max_pin: x.x
-  sqlite:
-    max_pin: x
```

Comment:
The removal of these `pin_run_as_build` packages will cause the same `qgis: error while loading shared libraries: libgdal.so.32` issue at https://github.com/conda-forge/qgis-feedstock/issues/276. See below on how to set these explicitly.